app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -270,7 +270,9 @@ public void insertBrowsingHistory() {
             return;
         } else if (SupportUtils.BLANK_URL.equals(url)) {
             return;
-        } else if (!UrlUtils.isSupportedProtocol(url)) {
+        }
+
+        if (!UrlUtils.isHttpOrHttps(url)) {
```

Comment:
This will block file:// is this expected behavior? Blocking it seems correct to me so r+ if this is not unexpected.

Reply:
Yes it is expected to block `file://` for now.